src/HonzaBotner.Discord.Services/VoiceManager.cs

```diff
@@ -160,7 +160,7 @@ private async Task DeleteAllUnusedVoiceChannelsAsync()
 
         private string? ConvertStringToValidState(string? input, string? defaultValue = null)
         {
-            input = Regex.Replace(input ?? "", @"[^\u0000-\u007F]+", string.Empty);
+            input = Regex.Replace(input ?? "", @"[^\u0000-\u007FěščřžýáíéťďňĚŠČŘŽÝÁÍÉŤĎŇ]+", string.Empty);

```

Comment:
Maybe we can use something like `[a-zA-Z0-9,.;:_$~'\s\-\+ěščřžýáíéóúůďťňĎŇŤŠČŘŽÝÁÍÉÚŮ]`?

Reply:
```suggestion
            input = Regex.Replace(input ?? "", @"[^a-zA-Z0-9,.;:_$~'\-+\sěščřžýáíéóúůďťňĎŇŤŠČŘŽÝÁÍÉÚŮ]+", string.Empty);

```

Reply:
I did some tests, @ostorc 

![image](https://user-images.githubusercontent.com/5287596/110028597-bb0e1b00-7d33-11eb-8922-ea04227e7a62.png)
![image](https://user-images.githubusercontent.com/5287596/110028605-bea1a200-7d33-11eb-9932-e6446bd91a3a.png)

Reply:
Nice. I was thinking about relaxing the validation more. We don't want to allow non-printable characters and some short list of Discord special characters (@, <, >, !). So who don't start from [this](https://stackoverflow.com/questions/40564692/c-sharp-regex-to-remove-non-printable-characters-and-control-characters-in-a/40568888)?